scripts/docker/nginx-plus/alpine/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG BASE_IMAGE
-FROM ${BASE_IMAGE} as install
```

Comment:
I'm not sure if this change is correct looking at it. If we want to pull containers from a different registry we should be able to. I think the BASE_IMAGE variable in the Makefile should contain a registry variable and you can optionally default it to docker.io/

Reply:
or an additional CONTAINER_REGISTRY variable is ok too